cybershell/scoring.py

```
@@ -0,0 +1,482 @@
+        estimated_value = base_value * multipliers
+
+        # Convert to 0-1 score
+        score = min(estimated_value / self.max_bounty, 1.0)
+
+        # Ensure minimum score for valid findings
+        if score < 0.1 and result.success:
+            score = 0.1
+
+        return score
+
+    def _identify_vuln_type(self, plugin_name: str, details: Dict) -> str:
```

Ruff (0.12.2)

`_identify_vuln_type` is too complex (11 > 10)

(C901)